.github/workflows/test.yml

```
@@ -24,7 +24,15 @@ jobs:
           # TODO: add jruby and truffleruby
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-${{ matrix.ruby }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
       - run: apt-get update && apt-get install -y nodejs # For execjs
       - name: bundle install
-        run: bundle install -j$(nproc) --retry 3
+        run: |
+          bundle config path vendor/bundle
+          bundle install -j$(nproc) --retry 3
```

```suggestion
        run: bundle install --path vendor/bundle -j$(nproc) --retry 3
```

Let me avoid increasing the number of lines

---

> [DEPRECATED] The `--path` flag is deprecated because it relies on being remembered across bundler invocations, which bundler will no longer do in future versions. Instead please use `bundle config set path 'vendor/bundle'`, and stop using this flag

Oh my... You know it better than me.